File: simpleloginjava/src/main/java/com/example/simpleloginjava/MainActivity.java
package com.example.simpleloginjava;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    Button login;
    EditText uname,pass;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        uname=(EditText)findViewById(R.id.etNum);
        pass=(EditText)findViewById(R.id.etPass);
        login=(Button)findViewById(R.id.btnok);
        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String nm=uname.getText().toString();
                String pas=pass.getText().toString();
                if(nm.equals(pas)){
                    Toast.makeText(MainActivity.this,"Login Successful",Toast.LENGTH_LONG).show();
                }
                else{
                    Toast.makeText(MainActivity.this,"Login UnSuccessful",Toast.LENGTH_LONG).show();

                }

            }
        });
    }
}

File: app/src/main/java/com/example/pc/simplelogingkotlin/MainActivity.kt
package com.example.pc.simplelogingkotlin

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        btnok.setOnClickListener{

            val no=etNum.text.toString()
            val pass=etPass.text.toString()
            if(no==pass) txtres.text="Successful"
            else txtres.text="Fail"
        }
    }
}